packages/jest-snapshot/src/printSnapshot.ts

```diff
@@ -6,6 +6,7 @@
  */
 
 import chalk = require('chalk');
+import {getObjectSubset} from 'expect/build/utils';
```

Comment:
should probably add a comment above this explaining why we're importing internal stuff from `expect`